### ISSUE TITLE
buscando dados do curso android com java

### ISSUE THREAD

**Issue by aironsg** [DevMasterTeam/Udemy-Andorid-Java#1]
sempre que clico nos links do curso, que são direcionados para este repósitorio,o link da pagina não encontra, ao verificar aqui, percebi que apenas os dados das aulas de java fundamentos estão em ordem. os demais foram apagados?

**Comment by santos-borges22** [comment]
Exatamente, somente dos primeiros módulos estão certos. Ou ele está atualizando, ou é algum erro ...
Estou tendo o mesmo problema
